### ISSUE TITLE
Wake-on-lan wakes host on client startup

### ISSUE THREAD

**Issue by shalak** [moonlight-stream/moonlight-qt#910]
**Describe the bug**
I have 3 hosts configured in my client, 1 is connected over LAN, the other 2 over WiFi. When I open the Moonlight client, the one on LAN immediately wakes-on-lan (even before I select the host to connect to)

**Steps to reproduce**
1. Open moonlight client
2. Observe the host

**Other Moonlight clients**
I don't have other clients.

**Moonlight settings (please complete the following information)**
Settings are default.

**Client PC details (please complete the following information)**
 - OS: MacOS 12.6 (21G115)
 - Moonlight Version:  4.3.1
 - GPU: Apple M1 Pro

**Server PC details (please complete the following information)**
 - OS: Windows 10 21H2

 - GeForce Experience version: 3.26.0.154]
 - Nvidia GPU driver: 527.37
 - Antivirus and firewall software: Windows Defender and Windows Firewall

**Moonlight Logs (please attach)**
[Moonlight.log](https://github.com/moonlight-stream/moonlight-qt/files/10184649/Moonlight.log)

**Additional context**
In Windows Event Viewer, the "Power-Troubleshooter" mentions that source of waking up is "Unknown"

**Comment by cgutman** [comment]
Moonlight only sends WoL magic packets when requested. You can confirm with Wireshark if you want to be sure.

Your host is probably configured to wake on _any_ traffic, not just magic packets. Make sure this checkbox is checked in the NIC properties.

![image](https://user-images.githubusercontent.com/2695644/206634364-fbf4a561-4bc8-4cf1-a354-f85b7dc531de.png)

**Comment by shalak** [comment]
Thank you!

I also noticed that only disabling the "Wake of Pattern Match" in Advanced Properties tab fixed the issue. I found a [document for Win7 describing possible patterns](https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-7/ee617165(v=ws.10)#overview-of-features) (couldn't find one for win10). So maybe it was NETBIOS name query or TCP connection attempt coming from the Moonlight client?

Anyway, the issue is no more :)
